libs/runtime/ebpf_platform.c

```diff
@@ -225,6 +225,18 @@ ebpf_query_time_since_boot(bool include_suspended_time)
     }
 }
 
+uint64_t
+ebpf_query_time_since_boot_approximate(bool include_suspend_time)
+{
+    LARGE_INTEGER time;
+    if (include_suspend_time) {
+        time = KeQueryInterruptTime();
+    } else {
+        time = KeQueryUnbiasedInterruptTime();
```

Comment:
This implementation would belong better in ebpf_platform_kernel.c than ebpf_platform.c

Reply:
Filed https://github.com/microsoft/ebpf-for-windows/issues/4005